app/routes/inbox/meetings.js

```diff
@@ -1,29 +1,34 @@
 import Route from '@ember/routing/route';
-import DataTableRouteMixin from 'ember-data-table/mixins/route';
+import { inject as service } from '@ember/service';
 
-export default class InboxMeetingsRoute extends Route.extend(
-  DataTableRouteMixin
-) {
-  modelName = 'zitting';
+export default class InboxMeetingsRoute extends Route {
+  @service store;
 
   queryParams = {
     page: { refreshModel: true },
     size: { refreshModel: true },
     sort: { refreshModel: true },
-    filter: { refreshModel: true },
-    // filter params
     title: { refreshModel: true },
```

Comment:
since there is no filtering on title, this query param can go

Reply:
fixed